.eslintrc

```diff
@@ -57,5 +57,13 @@
 
         "promise/always-return": "off",
         "promise/avoid-new": "off"
-    }
+    },
+    "overrides": [
+        {
+            "files": ["test/*.ts"],
+            "rules": {
+                "no-unused-expressions": "off"
```

Comment:
This rule doesn't work well with chai mocha expects like `expect(r.leverageAllocator).to.not.be.undefined`